### ISSUE TITLE
Changing coordinates with arrow keys don't work as expected.

### ISSUE THREAD

**Issue by Freestyle00** [vchelaru/FlatRedBall#203]
![AdFZGPc5gm](https://user-images.githubusercontent.com/69159664/134559761-78b5d77b-fc5a-4885-90d6-5cbebdb9938e.gif)
When trying to move the Sprite with the arrow keys it changes the coordinates but it doesn't move.
I logged the tasks:
```logs
8:04:06.5 - Removed Refreshing unreferenced files 0.046
8:04:07.2 - Created Reacting to changed file c:/users/nick_/documents/flatredballprojects/artyparty/ArtyParty/Entities
8:04:07.2 - Queued Reacting to changed file c:/users/nick_/documents/flatredballprojects/artyparty/ArtyParty/Entities
8:04:07.2 - Created Refreshing unreferenced files
8:04:07.2 - Started Reacting to changed file c:/users/nick_/documents/flatredballprojects/artyparty/ArtyParty/Entities
8:04:07.2 - Removed Reacting to changed file c:/users/nick_/documents/flatredballprojects/artyparty/ArtyParty/Entities 0.001
8:04:07.2 - Queued Refreshing unreferenced files
8:04:07.2 - Started Refreshing unreferenced files
8:04:07.2 - Removed Refreshing unreferenced files 0.036

```
Steps to reproduce.
1. Place a sprite 
2. Texture it 
3. Move it around with mouse ( this could be the issue )
4. Try to change its position with the arrow keys

**Comment by vchelaru** [comment]
I believe this bug is caused by the nudge code not checking of the object being nudged has a parent. It is probably changing the X and Y of the object, but it should check if it has a parent, and if so, change the RelativeX and RelativeY.

**Comment by vchelaru** [comment]
As expected, it was a Relative issue.

Fixed e562e75bb25442ffd0fbd8784a83bef5ca9fe334
